.github/workflows/CI.yml

```diff
@@ -125,6 +125,10 @@ jobs:
           apt update
           apt-get -y install python3-dev python3-venv python3-pybind11 ninja-build
 
+      - name: Install python packages
+        run: |
+          python3 -m pip install mpi4py --user
+
```

Comment:
maybe it's not needed to build the python bindings in this workflow as it's covered in the other 2? Otherwise, I guess the fix would be to create a venv.